frontend/server/src/DAO/Assignments.php

```diff
@@ -224,4 +224,22 @@ public static function unlinkProblemset(
             [$problemset->problemset_id]
         );
     }
+
+    public static function getNextPositionOrder(int $courseId): int {
+        $sql = '
+            SELECT
+                COUNT(a.assignment_id)
+            FROM
+                Assignments a
+            WHERE
+                a.course_id = ?;
+            ';
+
+        /** @var int|null */
+        $numberOfAssignments = \OmegaUp\MySQLConnection::getInstance()->GetOne(
+            $sql,
+            [$courseId]
+        );
+        return is_null($numberOfAssignments) ? 1 : $numberOfAssignments + 1;
```

Comment:
nit:

```suggestion
        return intval($numberOfAssignments) + 1;
```

esto debería ser equivalente.

Reply:
Listo